bin/tachyon-start.sh

```diff
@@ -26,6 +26,8 @@ MOPT is one of:
 
 -f  format Journal, UnderFS Data and Workers Folder on master
 
+-k  Do not try to kill prior running masters and/or workers in all or local
```

Comment:
-k seems to imply wanting to kill the running processes? Perhaps something like -no-kill, though its more verbose.